napari/_vispy/quaternion.py

```diff
@@ -16,13 +16,20 @@ def quaternion2euler(quaternion, degrees=False):
     angles : 3-tuple
         Euler angles in (rx, ry, rz) order.
     """
+    
+    def capper(value):
```

Comment:
Nice method!

What do you think of moving this outside of the function as a private one?

If you agree it would be nice to add a small test for it. Testing inner functions can be hard.

Reply:
Hey, thanks for the nice message ! :) I'm new here, as you might have noticed... I change the file a bit so that no inner function is needed, and it's still readable.

As it's my first time, I'm not used to the customs ; I hope it's still useful !